doc/onboarding.md

```diff
@@ -0,0 +1,85 @@
+# Numberscope onboarding
+
+## A note on commands
+
+Unless otherwise specified, all commands we ask you to run are supposed to be
+run in a Terminal emulator using a shell like Bash or Zsh (on Linux and macOS)
+or in Git Bash (on Windows).
+
+## Accounts, etc.
+
+(Steps 2 and 3 only apply to the members of the CU Boulder Experimental
+Mathematics Lab.)
+
+1. If you don't have an account, sign up for [GitHub](https://github.com).
+2. Make sure you've been added to the Numberscope GitHub organization.
+3. Make sure you've been added to the chat app (Slack or Zulip).
+
+## Setting up your computer for development
+
+1. If you don't have one, install a good text editor. Use whatever you want!
+   [Visual Studio Code](https://code.visualstudio.com/) is popular, and it
+   works well with our code base. If you want to use Visual Studio Code,
+   install the following extensions:
+
+    - Code Spell Checker
+    - TypeScript Vue Plugin (Volar)
+    - Vue Language Features (Volar)
+
+    For more info on extensions, see:
+    https://code.visualstudio.com/docs/editor/extension-marketplace.
+
+2. If you don't have [Git](https://git-scm.com/) installed, install it. We use
+   Git to keep track of the different versions of our code.
+3. If you don't have [NodeJS](https://nodejs.org/en/) installed, install it.
+   NodeJS allows you to run JavaScript outside of a web browser. We use it in
+   our front end code base.
+4. If you don't have [Python 3](https://www.python.org/) installed, install
+   it.
+5. Make sure you have a `venv` module by running the following command:
+    ```sh
+    python3 -m venv -h
+    ```
+    You should see help for the `venv` module. If you don't you might have to
+    install the module separately. How you do this depends on your system and
+    might require some googling.
+6. Somewhere on your computer, make a directory where you can keep Numberscope
+   code. I like to put a directory called `Code` in my home directory. You can
+   call this whatever you want.
+7. If you plan to submit your changes, you can "fork" (make your own copy of)
+   the repository:
```

Comment:
How about "If you plan to submit new code to become part of Numberscope at some time in the future, it is best to "fork"..."?  (Since they don't have changes yet and might think 'what changes'?)

Reply:
Addressed in https://github.com/numberscope/frontscope/pull/161/commits/d396fb6ea952263e47ea2fef852ad7f63a095da4.